### PR TITLE
Make testimonial dynamic on home page

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

File: coss/home/models.py [hunk 1]
```diff
@@ -36,3 +36,19 @@ class Meta:
 
     def __str__(self):
         return self.header_text
+
+
+@register_snippet
+class Testimonial(models.Model):
+    quote = RichTextField()
+    person = models.CharField(max_length=100)
+    photo = models.ImageField()
```
Comment (akatsoulas) [inline coss/home/models.py]: Doesn't this need an `upload_to` entry?

Comment (akatsoulas) [reply coss/home/models.py]: Can't we use here the image library we have as a foreign key?

File: coss/home/models.py [hunk 2]
```diff
@@ -36,3 +36,19 @@ class Meta:
 
     def __str__(self):
         return self.header_text
+
+
+@register_snippet
+class Testimonial(models.Model):
+    quote = RichTextField()
+    person = models.CharField(max_length=100)
+    photo = models.ImageField(uploaded_to='testimonials')
```
Comment (akatsoulas) [inline coss/home/models.py]: The more I think about it, the more I feel that we should have a ForeignKey here for the soon to come profiles. Thoughts? If you agree with this, I would suggest to block this PR until we have the profiles merged.

**Comment by comzeradd** [comment]
PR updated, using a `ForeignKey` to Image gallery as photo.

> The more I think about it, the more I feel that we should have a ForeignKey here for the soon to come profiles. Thoughts? If you agree with this, I would suggest to block this PR until we have the profiles merged.

My understanding is that the testimonials can also be people who don't necessarily have a profile on the website. For instance we can have a testimonial by a public figure.

**Pull request merged**